src/javascript/app/pages/cashier/payment_agent_withdraw.js

```diff
@@ -71,8 +71,23 @@ const PaymentAgentWithdraw = (() => {
 
             const form_id = `#${$(view_ids.form).find('form').attr('id')}`;
             const $form   = $(form_id);
-            const min     = getPaWithdrawalLimit(currency, 'min');
-            const max     = getPaWithdrawalLimit(currency, 'max');
+
+            const min  = () => {
+                const selected_val = $ddl_agents.val() || $txt_agents.val();
```

Comment:
why not use the function `getPALoginID`?

Reply:
Didn't notice the function 😄